qc_filtering/filter_functions.R

```diff
@@ -0,0 +1,124 @@
+## ---------------------------
+##
+## Script name: filter_functions.R
+##
+## Description:
```

Comment:
Maybe something you're planning to add later, but a description would be helpful for what functions within this script are doing

Reply:
Good idea, will do!